shared/oae/bundles/ui/default.properties

```diff
@@ -195,9 +198,68 @@ ACTIVITY_GROUP_UPDATE_MEMBER_ROLE_YOU = ${actor1Link} updated your role in the g
+ACTIVITY_INVITATION_ACCEPT_OTHER_OTHER_GROUP_1 = ${actor1Link} has accepted ${object1Link}'s invitation to the group &quot;${target1Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_OTHER_GROUP_2 = ${actor1Link} has accepted ${object1Link}'s invitation to the groups &quot;${target1Link}&quot; and &quot;${target2Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_OTHER_GROUP_2+ = ${actor1Link} has accepted ${object1Link}'s invitation to the group &quot;${target1Link}&quot; and ${targetCountMinusOne} others
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_COLLABDOC_1 = ${actor1Link} has accepted your invitation to the document &quot;${target1Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_FILE_1 = ${actor1Link} has accepted your invitation to the file &quot;${target1Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_LINK_1 = ${actor1Link} has accepted your invitation to the link &quot;${target1Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_CONTENT_2 = ${actor1Link} has accepted your invitation to collaborate on &quot;${target1Link}&quot; and &quot;${target2Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_CONTENT_2+ = ${actor1Link} has accepted your invitation to collaborate on &quot;${target1Link}&quot; and ${targetCountMinusOne} others
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_DISCUSSION_1 = ${actor1Link} has accepted your invitation to the discussion &quot;${target1Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_DISCUSSION_2 = ${actor1Link} has accepted your invitation to the discussions &quot;${target1Link}&quot; and &quot;${target2Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_DISCUSSION_2+ = ${actor1Link} has accepted your invitation to the discussion &quot;${target1Link}&quot; and ${targetCountMinusOne} others
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_FOLDER_1 = ${actor1Link} has accepted your invitation to the folder &quot;${target1Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_FOLDER_2 = ${actor1Link} has accepted your invitation to the folders &quot;${target1Link}&quot; and &quot;${target2Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_FOLDER_2+ = ${actor1Link} has accepted your invitation to the folder &quot;${target1Link}&quot; and ${targetCountMinusOne} others
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_GROUP_1 = ${actor1Link} has accepted your invitation to the group &quot;${target1Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_GROUP_2 = ${actor1Link} has accepted your invitation to the groups &quot;${target1Link}&quot; and &quot;${target2Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_OTHER_YOU_GROUP_2+ = ${actor1Link} has accepted your invitation to the group &quot;${target1Link}&quot; and ${targetCountMinusOne} others
+ACTIVITY_INVITATION_ACCEPT_YOU_OTHER_COLLABDOC_1 = You have accepted ${object1Link}'s invitation to the document &quot;${target1Link}&quot;
+ACTIVITY_INVITATION_ACCEPT_YOU_OTHER_FILE_1 = You have accepted ${object1Link}'s invitation to the file &quot;${target1Link}&quot;
```

Comment:
I don't think we can actually do `'s` for i18n reasons. For example, ending up with `You have accepted Nicolaas Matthijs's invitation ...` would be invalid English. We'll need to do something that always works like `You have accepted an invitation from Nicolaas Matthijs to ...`